core/src/epicli/cli/epicli.py

```diff
@@ -103,6 +129,24 @@ def run_init(args):
         initializer.run()
 
 
+def run_upgrade(args):
+    Config().output_dir = args.build_directory
+    with PatchEngine() as engine:
+        engine.run_upgrade(Config().output_dir)
+
+
+def run_backup(args):
+    Config().output_dir = args.build_directory
+    with PatchEngine() as engine:
+        engine.run_backup(Config().output_dir)
```

Comment:
I suggest not passing down the path since the whole idea of this global config is to access it throughout the CLI without passing down the same parameters all the time.